source/server/spongineer/incompatible.rst

```diff
@@ -52,6 +52,43 @@ Hammer Core
 
 - Solution: Change ``<entry key="World.ITickable.Override">true</entry>`` to ``false`` in the ``/asm/hammercore.xml`` file
 
+ICBM Classic
+~~~~~~~~~~~~
+- Versions: All
+- Problem: Missiles stop in midair and appear to lag in place.
+- Solution: Add the commented section to the ``sponge/global.conf`` entity-activation-range -> mods section
+
+.. code-block:: none
+
+    entity-activation-range {
+        # If 'true', newly discovered entities will be added to this config with a default value.
```

Comment:
Maybe we should remove the unrelated comments to make the copy comment more obvious.

Reply:
I concur. That was what I had intended with my prior comments; just give the section headers and relevant (new) bits, and cut out the rest of the fluff. 
If you want I can finish the editing Haxy, we don't mean to make contributors perform a formatting dance.

Reply:
Feel free to, I'm not the best at formatting things ;)